index.src.html

```diff
@@ -1103,11 +1103,19 @@ <h5 id="interfaces-credentialrequestoptions">
     <dd>
       If <code>true</code>, the user agent will only attempt to provide
       a {{Credential}} without user interaction. It MUST NOT present the user
-      with any visible prompt to grant access to a {{Credential}}: if the user
-      has opted-into always giving a particular site access to a particular set
-      of credentials, they will be provided. If not, the promise will resolve
-      with <code>undefined</code>. For processing details, see the algorithm
-      defined in [[#request-credential]].
+      with any visible prompt to grant access to a {{Credential}} or to select
+      a {{Credential}}: if the user has opted-into always giving a particular
+      site access to a particular credential, it will be provided. If not, the
+      promise will resolve with <code>undefined</code>.
+
+      If <code>false</code>, the user agent MAY ask the user to grant access to
+      or select a {{Credential}}. If the user has opted-into always giving a
+      particular site access to a particular credential, it MUST be provided
```

Comment:
s/MUST/will/ I don't think you're laying out normative requirements here for returning specific credentials: you're describing the impact of the requirements in https://w3c.github.io/webappsec-credential-management/#request-credential (which calls out to grab unmediated credentials first, then to mediated credentials).